src/module/webauthn-validator/types.ts

```diff
@@ -0,0 +1,24 @@
+import { Address, type Hex } from 'viem'
+
+export type WebauthnCredential = {
```

Comment:
could you export these types to the top level (ie through index in webauthn-validator etc)